src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.cs

```diff
@@ -769,7 +769,7 @@ public bool IsPrefix(string source, string prefix, CompareOptions options)
         /// <exception cref="ArgumentException">
         /// <paramref name="options"/> contains an unsupported combination of flags.
         /// </exception>
-        public bool IsPrefix(ReadOnlySpan<char> source, ReadOnlySpan<char> prefix, CompareOptions options = CompareOptions.None)
+        public unsafe bool IsPrefix(ReadOnlySpan<char> source, ReadOnlySpan<char> prefix, CompareOptions options = CompareOptions.None)
```

Comment:
what is our current guidance on making existing public methods unsafe?

I would assume that if something does not use pointers as arguments|restult then it should not be unsafe? This applies to this and other methods in this type

Reply:
"unsafe" doesn't change the signature, it's only an implementation detail. Callers aren't required to use unsafe blocks when calling unless the arguments expose pointers (which these don't).